features/03_testing_frameworks/cucumber/steps/filesystem/check_file_content.feature

```diff
@@ -3,6 +3,63 @@ Feature: Check file content
   Background:
     Given I use a fixture named "cli-app"
 
+  Scenario: Check file contents with plain text
+    Given a file named "features/non-existence.feature" with:
```

Comment:
why is this file called non-existence?

Reply:
Not sure. I copied this feature from master I think.